my-app/src/App.js

```diff
@@ -1,14 +1,17 @@
 import Header from "./components/Header/Header";
 import Card from "./components/Card/Card";
 import "./App.css";
-import "./data/mock";
+import mock from "./data/mock";
 
-function App(mock) {
+function App() {
   return (
     <div className="App">
```

Comment:
it is better to follow same codestyle for names (classname in this case) 
(first letter to lowercase)